vmdb/lib/vmdb/configuration_encoder.rb

```diff
@@ -19,6 +19,23 @@ def self.symbolize!(h)
       h.each_key { |k| h[k].symbolize_keys! }.symbolize_keys!
     end
 
+    class YamlLoader < Psych::Visitors::ToRuby
+      NUMBER_WITH_METHOD_REGEX = /^([0-9\.,]+)\.([a-z]+)$/
+
+      def visit_Psych_Nodes_Scalar(node)
+        if node.value =~ NUMBER_WITH_METHOD_REGEX
+          if $2 == 'percent'
+            super
```

Comment:
Is this right?  The YAML file will have something like `"1.percent"`, so the super call here will pass the full string up.  I think it should be `$1.to_i` here.